platform/product-management/release-plan-template.md

```diff
@@ -6,6 +6,38 @@ So: **how might we craft a release plan to test our product "in the wild" at a s
 
 That's what this Release Plan Template is for! And note - there are feature toggles and beta banners at your disposal that you can use as a part of your plan.
 
+## Do I need unmoderated production testing?
+
+**Yes**, staged rollout is required unless you can confidently answer "yes" to all of the following:
+
+* This change does not add substantial new functionality to VA.gov
+* This change does not impact user flows through tasks
+* This change does not affect traffic to backend services
+
+*Example*: a change to a page's text content **could skip** staged rollout
+
+*Example*: a visual redesign to a page that doesn't affect user flows **could skip** staged rollout
```

Comment:
Suggest: a *minor* visual redesign? Or another qualifier to clarify that if you're changing an entire page design you should do a phased rollout (IMO)